packages/filesystem/src/common/filesystem.ts

```diff
@@ -23,7 +23,7 @@ export interface FileSystem extends Disposable {
     getFileStat(uri: string): Promise<FileStat>;
 
     /**
-     *Finds out if a file identified by the resource exists.
+     * Finds out if a file identified by the resource exists.
```

Comment:
:+1: I did the same on my branch, and have seen in other PRs too. You have to push ASAP, otherwise, you will have the conflicts :)